openfisca_us/variables/gov/states/mo/tax/income/credits/mo_property_tax_credit/mo_property_tax_credit_demographic_tests.py

```diff
@@ -0,0 +1,37 @@
+from openfisca_us.model_api import *
+
+
+class mo_property_tax_credit_demographic_tests(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MO property tax credit demographic eligiblity test"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://dor.mo.gov/forms/MO-PTS_2021.pdf"
+    defined_for = StateCode.MO
+
+    def formula(tax_unit, period, parameters):
+        ##Eligibility
+        #vars for age test
+        age_head = tax_unit("age_head", period)
+        age_spouse = tax_unit("age_spouse", period)
+        head_age_test = age_head >= 65
+        spouse_age_test = age_spouse >= 65
+        age_test = (head_age_test + spouse_age_test) >= 1
```

Comment:
Similarly below to make variables more self-documenting
```suggestion
        elderly_head_or_spouse = elderly_head | elderly_spouse
```